Description: Change default search path for game data /usr/games/doom3
Author: Tobias Frost <tobi@debian.org>
Last-Update: 2015-02-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/neo/CMakeLists.txt
+++ b/neo/CMakeLists.txt
@@ -269,7 +269,7 @@
 
 set(bindir		"${CMAKE_INSTALL_FULL_BINDIR}")
 set(libdir		"${CMAKE_INSTALL_FULL_LIBDIR}/dhewm3")
-set(datadir		"${CMAKE_INSTALL_FULL_DATADIR}/dhewm3")
+set(datadir		"${CMAKE_INSTALL_FULL_DATADIR}/doom3")
 
 configure_file(
 	"${CMAKE_SOURCE_DIR}/config.h.in"
